content/docs/user-guide/basic-concepts/file-link.md

```diff
@@ -0,0 +1,10 @@
+---
+name: File Linking
+match:
+  [link, symlink, hardlink, reflink, linked, symlinked, hardlinked, reflinked]
```

Comment:
We have a couple issues related to adding and reorganizing concepts. Mainly #550 but also #1395. Would you like to take on one of those at least partially?

Reply:
Yes I think that's something I should do later this week/next week but it looks like they're already assigned to @iesahin.

Reply:
@casperdcl Any improvements, edits etc. are all welcome. You can take over the basic concepts section (and related issues) completely or update/edit the tasks in #550. I already have too much on my plate. Thank you very much.

Reply:
@iesahin please un-assign from anything you're no longer actively involved with 🙏 Thanks!